### PR TITLE
To Do list: list structure

### PR DESCRIPTION
**Pull request by vabuyia**
>>Tasks have been rendered according to the values. 
>>Project style has been changed.

Note: Linters are not working on the Github BUT I have run linters locally and the file has passed. Kindly do the same.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,4 +1,47 @@
...
+  },
+  {
+    index: 2,
+    description: 'Visit Friends and Family',
+    completed: false,
+  },
+  {
+    index: 1,
+    description: 'Going home',
+    completed: false,
```
Comment (Gambit142) [inline src/index.js:17]: - [ ] Kindly render the tasks according to the `index` values in the array. That means that _**Going to the gym**_ should be the first, _**Going home**_ should be the second and _**Visit Friends and Family**_ should be the third based on the values of the `index`.

File: src/style.css [hunk 2]
```diff
@@ -1,3 +1,112 @@
...
 body {
-    background-color: bisque;
-}
\ No newline at end of file
+  height: 100vh;
+  width: 100%;
+  background: #dfdad9;
+  font-family: 'poppins', Arial, Helvetica, sans-serif;
+  display: flex;
+}
```
Comment (Gambit142) [inline src/style.css]: - [ ] Kindly style the application according to the template used in this [video](https://www.youtube.com/watch?v=AcUd-_Yjjqg)

### Template 

image

### Your Design

image

**Review by Gambit142 — CHANGES_REQUESTED** [review]
Hi @vabuyia 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by elisha2kyakpo1 — APPROVED** [review]
Hi 👋 @vabuyia, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**